### ISSUE TITLE
Latest BeardedSpice doesn't recognize Web spotify

### ISSUE THREAD

**Issue by Nuruddinjr** [beardedspice/beardedspice#777]
On Latest BeardedSpice and Chrome version I can't see Spotify playing, youtube is working though.

Please, advise.
http://take.ms/cjM49

**Comment by Stillness-2** [comment]
If you have a time,  please test this: https://github.com/beardedspice/beardedspice/issues/771#issuecomment-376469908
:)

**Comment by Nuruddinjr** [comment]
It doesn't work, installed extension, and debug version, but won't work.

**Comment by Nuruddinjr** [comment]
@Stillness-2 yes I could connect it, on youtube I can see what track is playing and can stop and play next, Spotify web doesn't work

**Comment by Starfight** [comment]
Same problem here :

**Comment by Starfight** [comment]
Same problem here:
Chrome version: 69.0.3497.92 (64 bits)
Beardedspice version: 2.2.3 (latest)
macOS: High Sierra 10.13.6

Others website like netflix or youtube work very well after enable AppleScript event and restart both BS and Chrome but Spotify does not 😢 
If I can help you debugging, what can I test ?

**Comment by AAverin** [comment]
Still have this problem

**Comment by NiklasWebnuts** [comment]
@devinrhode2 Thanks for looking into this. Even with "allow javascript from apple events" toggled on, this still happens. I suspect that the issue might be lying somewhere on spotifys end (maybe). I don't know for sure. I'm not using the web client that much these days
